core/src/main/scala/cats/data/XorT.scala

```diff
@@ -51,6 +51,8 @@ final case class XorT[F[_], A, B](value: F[A Xor B]) {
 
   def exists(f: B => Boolean)(implicit F: Functor[F]): F[Boolean] = F.map(value)(_.exists(f))
 
+  def ensure[AA >: A](ifLeft: => AA)(f: B => Boolean)(implicit F: Functor[F]): XorT[F, AA, B] = XorT(F.map(value)(_.ensure(ifLeft)(f)))
```

Comment:
The name `ifLeft` surprised me. This is the value that is used if the input is a `Right` but the predicate fails, right? I see that you are just following the name used in `Xor`, but I'm thinking we should probably change both of these to `ifFalse`, `whenFalse`, `predicateFailed` or something like that. What do you think?